packages/linux/package.mk

```diff
@@ -170,7 +170,11 @@ pre_make_target() {
     ${PKG_BUILD}/scripts/config --set-str CONFIG_EXTRA_FIRMWARE_DIR "external-firmware"
   fi
 
-  kernel_make oldconfig
+  NEW_KERNEL_OPTIONS=$(kernel_make -s listnewconfig)
+  if [ -n "${NEW_KERNEL_OPTIONS}" ]; then
+      print_color CLR_WARNING "LINUX: new kernel options found: \n${NEW_KERNEL_OPTIONS%%}\nPlease run ./tools/check_kernel_config\n"
+  fi
+  yes "" | kernel_make oldconfig > /dev/null
```

Comment:
We probably want the option default not "yes" for everything.

Reply:
Why not just use olddefconfig?

Reply:
I thought about that as well but there are subtle differences in how kconfig processes defconfig/oldconfig/olddefconfig.

eg oldconfig will restart the config process if it detects a new option, olddefconfig does not. This can lead to different results depending on the kconfig order (eg if A depends on B but B is after A in kconfig).

As I'm not 100% sure if this might cause subtle issues I decided against switching to olddefconfig and sticking to oldconfig.